artifactory.py

```diff
@@ -63,6 +64,8 @@
     import ConfigParser as configparser
 
 default_config_path = "~/.artifactory_python.cfg"
+if platform.system().lower() == "windows":
```

Comment:
Let's check real values without converting them https://docs.python.org/3/library/platform.html#platform.system
```suggestion
if platform.system() == "Windows":
```